packages/story-editor/src/components/mediaRecording/settingsModal.js

```diff
@@ -169,6 +196,18 @@ function SettingsModal() {
           />
         </Row>
       )}
+      {videoInputs.length > 0 && (
```

Comment:
Now that we have the quick action I think we can remove this from the settings modal again

Reply:
I asked about it on design sync, Andrew was in favor of leaving it, in the future I guess we will add other options like virtual background, etc.

The other thing that is worth considering is the fact that none of the quick actions are persisted - if you disable audio or video it will be enabled in the next session and so does the blur effect, but if you change it in the settings - it will be persisted (like other settings options), not sure if it's a bug or feature 😉